### PR TITLE
custom guild support

### PR DESCRIPTION
**Pull request by narze**
- Add new guilds
- Refactor getName, getSlug
- Remove pattaya 😛 
- Add link to twitter post about guild creation (You can move elsewhere)
image

### CODE REVIEW

**Comment by narze** [comment]
@scomma Note : merge after guild is created

File: src/lib/guilds.ts [hunk 1]
```diff
@@ -76,5 +95,43 @@ export const guilds = [
...
+    emoji: '😎',
+    slug: '9tun',
+  },
+  {
+    // 80
+    custom: true,
+    th: 'Onlyfans TH',
+    emoji: '💸',
+    slug: 'onlyfans_th',
+  },
```
Comment (scomma) [inline src/lib/guilds.ts:118]: these should later be supplied by the server?

**Review by scomma — APPROVED** [review]
👍

File: src/lib/guilds.ts (hunk 1 above)
Comment (narze) [reply src/lib/guilds.ts:118]: Not sure if it works since if user uses link prayut.click?g=slug, it won't get geolocation

**Pull request merged**